vunit/sim_if/rivierapro.py

```diff
@@ -284,15 +284,13 @@ def _create_load_function(
 
         tcl = """
 proc vunit_load {{}} {{
-    # Make the variable 'aldec' visible; otherwise, the Matlab interface
-    # is broken because vsim does not find the library aldec_matlab_cosim.
-    global aldec
-    # Make the variable 'LICENSE_QUEUE' visible (if set); otherwise vsim
-    # will not wait for simulation licenses.
-    global LICENSE_QUEUE
-
+    # Run the 'vsim' command in the global variable context. This will make
```

Comment:
I feel this paragraph is now harder to understand than https://github.com/VUnit/vunit/blame/2bb4d331b629896d3941733454d533075831222a/vunit/sim_if/rivierapro.py#L290. I'd suggest:

```
# ... Otherwise:
# - The Matlab interface is broken because vsim does not find the
#   library aldec_matlab_cosim.
# - vsim will not wait for simulation licenses.
```

Reply:
That is more readable. I have made the changes.